exampleSite/hugo.yaml

```diff
@@ -38,3 +38,17 @@ markup:
   goldmark:
     renderer:
       unsafe: true
+    extensions:
+      passthrough:
+        enable: true
+        delimiters:
+          block:
+          - - \[
+            - \]
+          - - $$
+            - $$
+          inline:
+          - - \(
+            - \)
+          - - $
+            - $
```

Comment:
I'd prefer not enabling `$ $` for inline math and restrict the delimiters to `\( \)`.

```suggestion
```